File: 1421-find-numbers-with-even-number-of-digits/find-numbers-with-even-number-of-digits.cpp
class Solution {
public:
    int findNumbers(vector<int>& nums) {
        int n=nums.size();
        int i=0;
        int count=0;
        while(i<n){
            int digit= nums[i];
            int d=0;
            while(digit>0){
                digit /= 10;
                d++;
                
            }
            if(d%2==0){
                count++;
            }
            i++;
        }
        return count;
    }
};

File: 1421-find-numbers-with-even-number-of-digits/find-numbers-with-even-number-of-digits.java
class Solution {
    int count(int n){
        int digit=0;
        while(n>0){
          n= n/10;
          digit++;
        }
        return digit;
    }
    public int findNumbers(int[] nums) {
        int n= nums.length;
        int ans=0;
        for(int i=0 ; i< n; i++){
            if(count(nums[i])% 2==0){ ans++;}
        }

        return ans;
    }
}